### PR TITLE
Show the number of entered results in the events overview

### LINKED ISSUE #228: Overview for scoretaking progress
In admin view being able to get an overview over how many results have been entered in each round could be helpful. 

Perhaps something like this:
image

This would help getting an overview of what is done and what is missing. 

Alternatively some sort of indicator on whether a round is fully entered or not.

### PR DESCRIPTION
**Pull request by maxidragon**
Closes #228 

Not sure how to style this - any ideas? It looks like this:

image

### CODE REVIEW

**Comment by jonatanklosko** [comment]
I updated the design slightly:

image

We already use the "N of M entered" wording in the round view, so I used the same for consistency. I think it looks fine, but we can always revisit.

File: client/src/components/admin/AdminCompetitionEvents/AdminCompetitionEvents.jsx [hunk 1]
```diff
@@ -21,6 +21,8 @@ const COMPETITION_EVENTS_QUERY = gql`
           name
           open
           finished
+          enteredResults
+          totalResults
```
Comment (jonatanklosko) [inline client/src/components/admin/AdminCompetitionEvents/AdminCompetitionEvents.jsx]: Reading this names without context one may assume the values is a list of results (technically we don't have a nested `{}`, but that's a detail). I would make the naming more explicit, as in `numEnteredResults` and `numResults`.

File: client/src/components/admin/AdminCompetitionEvents/ClearRoundButton.jsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ const CLEAR_ROUND_MUTATION = gql`
       round {
         id
         open
+        enteredResults
+        totalResults
```
Comment (jonatanklosko) [inline client/src/components/admin/AdminCompetitionEvents/ClearRoundButton.jsx]: I added this for both open and clear. The reason for including particular fields in the graphql response is that the local cache and therefore existing queries are immediately updated. Without this, after opening a round we would see `0 of 0 entered` :)

**Comment by maxidragon** [comment]
It looks much better!

File: lib/wca_live/scoretaking/round.ex [hunk 3]
```diff
@@ -134,6 +134,23 @@ defmodule WcaLive.Scoretaking.Round do
...
+
+  def entered_results(round) do
+    format = Format.get_by_id!(round.format_id)
+
+    entered_results =
+      round.results
+      |> Enum.filter(&Result.has_expected_attempts?(&1, format.number_of_attempts, round.cutoff))
+      |> length()
+
+    entered_results
```
Comment (jonatanklosko) [inline lib/wca_live/scoretaking/round.ex]:
```suggestion
    
    Enum.count(round.results, &Result.has_expected_attempts?(&1, format.number_of_attempts, round.cutoff))
```

File: lib/wca_live/scoretaking/round.ex [hunk 4]
```diff
@@ -134,6 +134,23 @@ defmodule WcaLive.Scoretaking.Round do
...
+  end
+
+  def total_results(round) do
+    length(round.results)
+  end
```
Comment (jonatanklosko) [inline lib/wca_live/scoretaking/round.ex]: For this one, I would just do `length(round.results)` directly :)

```suggestion
```

File: lib/wca_live/scoretaking/round.ex [hunk 5]
```diff
@@ -134,6 +134,17 @@ defmodule WcaLive.Scoretaking.Round do
       (unfinished_results / total_results < 0.1 and not active?(round))
   end
 
+  def num_entered_results(%Round{results: []}), do: 0
+
+  def num_entered_results(round) do
```
Comment (jonatanklosko) [inline lib/wca_live/scoretaking/round.ex]:
```suggestion
  @doc """
  Returns the number of results that have all the expected attempts entered.
  """
  @spec num_entered_results(%Round{}) :: non_neg_integer()
  def num_entered_results(round) do
```

**Review by jonatanklosko — APPROVED** [review]
Thanks!

**Pull request merged**